specification/electronic-prescription-service-api.yaml

```diff
@@ -132,26 +132,32 @@ info:
     ## Responses
     ### 2xx responses
     **Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
-    | Endpoint              | HTTP code | Response format                                                                                                                                             |
-    | --------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
-    | POST /$prepare        | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
-    | POST /$process-message| 200       | [See /$process-message response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-sendPrescription) |
+    
+    | Endpoint               | HTTP code | Response format                                                                                                                                             |
+    | ---------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | POST /$prepare         | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
+    | POST /$process-message | 200       | [See /$process-message response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-sendPrescription) |
+    
     ### 4xx responses
     **Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
+    
     | Endpoint              | HTTP code | Response format                           | Environment      | Cause                 |
     | --------------------- | --------- | ----------------------------------------- | ---------------- | --------------------- |
     | Any                   | 400       | See "Unsupported operation" example below | Sandbox          | Unsupported operation |
     | Any                   | 408       | Returns original request data             | Integration test | Request timeout       |
```

Comment:
Isn't the "int" environment is just called "Integration" rather everywhere else?